go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/greenplum-db/gp-common-go-libs
 
-go 1.21
+go 1.23
+
+toolchain go1.24.2
```

Comment:
Why we need set toolchain to go1.24.2?

Reply:
It comes, automatically once we run `go mod tidy` after updating the go.mod file.

Reply:
But if I delete it, it will not be added automatically by `go mod tidy`.

Reply:
Yes, it does I like this. I also tried deleting it.
Initially when we make change, and run `go mod tidy`, it is added but if we delete it and then run `go mod tidy`, it usually  does not get added. 
So I did not remove it this time. But I am fine both ways, I will remove it and push the changes.